pkg/cache/internal/informers_map.go

```diff
@@ -145,7 +152,7 @@ type specificInformersMap struct {
 
 // Start calls Run on each of the informers and sets started to true.  Blocks on the context.
 // It doesn't return start because it can't return an error, and it's not a runnable directly.
-func (ip *specificInformersMap) Start(ctx context.Context) {
+func (ip *InformersMap) Start(ctx context.Context) error {
```

Comment:
Q: Was the error return parameter just added in case we want to return errors in the future? (as of now we only return nil)

Reply:
It was added to satisfy the `Runnable` interface, the same (nil error returned in any case) was before in the deleg_map file

Reply:
Makes sense, thx for the info!